fili-core/src/main/java/com/yahoo/bard/webservice/async/ResponseException.java

```diff
@@ -64,7 +95,29 @@ public ResponseException(
             DruidQuery<?> druidQuery,
             Throwable cause
     ) {
-        super(buildMessage(reason, description, statusCode, druidQuery, cause));
+        this(statusCode, reason, description, druidQuery, cause, new ObjectMapper().writer());
+    }
```

Comment:
And by that token, probably deprecate other constructors to this thing that don't take in the writer they'll need as well.